### DISCUSSION TITLE
Safety

### DISCUSSION THREAD

**Discussion by FedorMatvienko** [community/community#28984] [Other Feature Feedback, Questions, & Ideas]
Someone deleted my ssh key without my knowledge. When hovering over the icon, the name is not displayed. Who is it? And why was my ssh deleted?
![взлом](https://user-images.githubusercontent.com/65041266/183848101-57c58554-746b-4e75-9233-f608c6a7bace.png)

**Comment by airtower-luna** [comment]
Could it be because the key hasn't been used in a long time? See: https://docs.github.com/en/authentication/troubleshooting-ssh/deleted-or-missing-ssh-keys
